.gitignore

```diff
@@ -12,7 +12,7 @@ aclocal.m4
 ar-lib
 autom4te.cache/
 coap_config.h
-/coap_config.h.in
+coap_config.h.in
```

Comment:
As `coap_config.h.in` should be in the root directory, this change is not really necessary (although it does not hurt, either).

Hower, the new `cmake_coap_config.h.in` is missing.

Reply:
`coap_config.h.in` is generated as a template file by `autogen.sh` keyed off `configure.ac` `(AC_CONFIG_HEADERS([coap_config.h]))` and is not a git controlled file - hence exclusion.  Previously `cmake_coap_config.h.in` was `include/coap2/coap_config.h` - I did not like the name clash/confusion - hence file name change.
.
`cmake_coap_config.h.in` is a git controlled file and so, correctly, is not missing from `.gitignore`.

Reply:
Oh, okay. For some reason, I was thinking that this file was generated automatically.